src/registrar/migrations/0064_domain_domain_notes_domainapplication_notes_and_more.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 4.2.7 on 2024-01-25 16:18
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("registrar", "0063_veryimportantperson"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="domain",
+            name="domain_notes",
```

Comment:
**suggestion (non-blocking)**: I think that we should be consistent and use the name "notes" for all of these fields. It isn't clear why the field for the `Domain` model needs to have a different name, necessitating a future developer to remember that this one is different.

Reply:
Yeah thats totally reasonable, great suggestion. The reasoning for the naming difference is because Domain has an inline "DomainInformation" object, and both of those have a "notes" field so I figured it needed a degree of clarity, but the help text and the inline is more than sufficient to differentiate it